modules/cas_cache/classifier_defs.h

```diff
@@ -118,6 +118,12 @@ struct cas_cls_numeric {
 	uint64_t v_u64;
 };
 
+/* String condition context */
+struct cas_cls_string {
```

Comment:
Alternatively you can just store the bare string in classifier context instead of wrapping into a struct.

Reply:
The structure name carries useful information about the data while bare string is pretty generic, so it's not that bad. However currently it costs us two allocations - one for structure and one for string, which may not be a big issue in the management path, but anyway it complicates error handling/cleanup. Maybe instead of storing pointer we can store fixed size buffer inside the structure?